src/applications/debt-letters/components/OnThisPageLinks.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 export const OnThisPageLinks = ({ isDetailsPage, hasHistory }) => (
   <div>
-    <h2 className="vads-u-font-size--h3">On this page</h2>
+    <h2>On this page</h2>
```

Comment:
Have you tried the [`<va-on-this-page>`](https://design.va.gov/storybook/?path=/docs/components-va-on-this-page--default) web component?

Reply:
yep .. this component was created before we had an actual design component for this and will be replaced at a later time